source/core/TimeFunctions.cpp

```diff
@@ -166,11 +166,14 @@ namespace Vireo {
     }
 
     //------------------------------------------------------------
-    VIREO_FUNCTION_SIGNATURE2(DateTimeToTimestamp, LVDateTimeRec, Timestamp)
+    VIREO_FUNCTION_SIGNATURE3(DateTimeToTimestamp, LVDateTimeRec, Boolean, Timestamp)
     {
         LVDateTimeRec *dt = _ParamPointer(0);
+        Boolean isUTC = _ParamPointer(1) ? _Param(1) : false;
+        Int32 timeZoneOffset = isUTC ? 0 : Date::getLocaletimeZone();
         Timestamp timestamp(dt->fractional_secs, dt->second, dt->minute, dt->hour, dt->day_of_month, dt->month, dt->year);
-        _Param(1) = timestamp;
+        Timestamp local = timestamp - timeZoneOffset;
```

Comment:
I mildly disagree with this.  I think overloading + is well-defined to operate only on the _Integer portion.